Index: Sourcebook Homebrews/Feats/Dirty-Fighting.md
===================================================================
diff --git a/Sourcebook Homebrews/Feats/Dirty-Fighting.md b/Sourcebook Homebrews/Feats/Dirty-Fighting.md
deleted file mode 100644
--- a/Sourcebook Homebrews/Feats/Dirty-Fighting.md	
+++ /dev/null	
@@ -1,16 +0,0 @@
----
-aliases: ["Dirty Fighting"]
-description: 
-permalink: 
-publish: true
----
-
-> [!skill] Dirty Fighting 🍻
-> **Details**:: Add a Crit Die to the 2-7 Accuracy Range for all Favored Weapons.
-> ### Metadata
-> **Source**:: [Scout's B&B DLC 3: Into the Wonderlands](https://docs.google.com/document/d/1MLOgrWwcLNTnP9PuXrKiLImy7SUh4hXO8arVUAlmdp0/edit).
-> **Tags**:: #BnB/Homebrew/Feats/Assassin #BnB/Homebrew/Scout/DLC3
-> **Tree**:: [[Compendium/BnB/Sourcebook Homebrews/Classes/Assassin Stabbomancer|Stabbomancer Assassin]].
-
-
-
